php-fpm/recipes/cloudwatch.rb

```diff
@@ -0,0 +1,22 @@
+cronscript = '/opt/easybib/bin/phpfpm-cloudwatch.sh'
```

Comment:
In lieu of us maybe using another launchpad repo _soon_, maybe we should use the prefix here? I think it's `node['php-fpm']['prefix']`. Or we settle on something like, `/usr/local/bin` in general for things _we_ install/maintain.